docs/workshop-schedule.md

```diff
@@ -3,18 +3,26 @@ title: Workshop Schedule
 nav_title: Schedule
 ---
 
-| Time | Topic | 
+Below is the schedule for the Data Lab Reproducible Research Practices Workshop on June 10, 2022.
+
+Each topic is linked to a Google Slides document with the content we plan to present.
+
+| Time | Topic |
 |------|---------|
 | 9:00 - 10:15 am  | [Welcome](https://docs.google.com/presentation/d/1GgqWl6las6ekQtKLmPzJ8UD_cftIbJjDTS2gJpStcMQ/edit?usp=sharing) <br> [Why does reproducibility matter?](https://docs.google.com/presentation/d/1qfulAR4jD0KS7NfrLHpwT6SWl-7APBmqNnAwGpXX5oo/edit?usp=sharing) |
-| 10:15 - 10:45 am | [Project Organization](https://docs.google.com/presentation/d/1ncqxXlC0-PGEK-yE7S-nDYnMPhrOUPbI95EJy283wCs/edit?usp=sharing) |
+| 10:15 - 10:45 am | [Project organization](https://docs.google.com/presentation/d/1ncqxXlC0-PGEK-yE7S-nDYnMPhrOUPbI95EJy283wCs/edit?usp=sharing) |
```

Comment:
The slides have this title (and other "harmonizations") in all title case. If we want the schedule and slide titles to match, we will have to either make the schedule titles title case, or change the slides to match what is now in the schedule. I'm ok with any direction here if you have a preference.

Reply:
Since the slides titles aren't all on the same page, the lack of consistency bothers me less, and I doubt anyone will notice. We could change them, but I don't think they don't have to match. (See also citation formats vs. paper titles)

Reply:
> I don't think they don't have to match 
I agree, just wasn't sure about your desired "harmonization" level!